Generator.Application/Validations/PayloadValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using Generator.Application.DTOs;
+
+namespace Generator.Application.Validations
+{
+    public class PayloadValidator : AbstractValidator<PayloadDto>
+    {
+        public PayloadValidator()
+        {
+            RuleFor(p => p.PictureA).NotEmpty();
+            RuleFor(p => p.PictureA.ToString()).Length(36).Must(p => p[8] == '-' && p[13] == '-' && p[18] == '-' && p[23] == '-');
```

Comment:
to tu nie powinno być. asp net core samemu da radę zbindowac coś do Guidu, nie musisz mu dawac stringu. Jeśli nie bedzie w stanie zrobic binda do Guid, wtedy wyjebie model state invalid => w [ApiController] wywali się automatycznie 400 Bad Request

Reply:
nvm, usunales juz